client/source/js/modules/analysis/optimization.html

```diff
@@ -1,540 +1,507 @@
 <div class="page">
```

Comment:
Please use spaces instead of tabs - as spaces are already used all over the project